src/Config.js

```diff
@@ -133,9 +204,20 @@ export class Config {
     this.validateRateLimit(rateLimit);
     this.validateLogLevels(logLevels);
     this.validateDatabaseOptions(databaseOptions);
+    this.validateCustomPages(customPages);
     this.validateAllowClientClassCreation(allowClientClassCreation);
   }
 
+  static validateCustomPages(customPages) {
+    if (!customPages) return;
+
+    if (Object.prototype.toString.call(customPages) !== '[object Object]') {
+      throw Error('Parse Server option customPages must be an object.');
+    }
+
+    Config.validateConfigKeyNames(Object.keys(CustomPagesOptions), Object.keys(customPages), 'customPages');
```

Comment:
```suggestion
    Config.validateConfigKeyNames(Object.keys(CustomPagesOptions), Object.keys(customPages), 'customPages');
```

Why do we need to pass `Object.keys(CustomPagesOptions)` and `Object.keys(customPages)`? Can't we just pass `CustomPagesOptions` and `customPages` and then get the `Object.keys(...)` from them inside the `Config.validateConfigKeyNames` method? That would remove a lot of duplicate code.